### PR TITLE
Modify data_backed_service module.md file

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Modify data_backed_service module.md file

### CODE REVIEW

File: docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md [hunk 1]
```diff
@@ -1,87 +1,35 @@
-Guide on Database-Backed Services using Ballerina.
+Template for Database-Backed Services using Ballerina
 
-# Guide Overview
+# Database-Backed Services using BallerinaP
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md]:
```suggestion
# Database-backed Service
```

File: docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md [hunk 2]
```diff
@@ -1,87 +1,35 @@
-Guide on Database-Backed Services using Ballerina.
+Template for Database-Backed Services using Ballerina
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md]:
```suggestion
Template for Database-backed Service
```

File: docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md (hunk 1 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md]: Fixed

File: docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/data-backed-service/src/data_backed_service/Module.md]: Fixed

**Pull request merged**